cinema_service/settings.py

```diff
@@ -133,8 +143,9 @@
 
 STATIC_URL = "static/"
 
+MEDIA_ROOT = "/files/media"
```

Comment:
The `MEDIA_ROOT` is set to a fixed path. Consider using `os.path.join(BASE_DIR, 'media')` to ensure the media files are stored relative to the project directory.